lnd/config.go

```diff
@@ -22,6 +22,14 @@ type Config struct {
 	LNDClusterPubkeys            string  `envconfig:"LND_CLUSTER_PUBKEYS"` //comma-seperated list of public keys of the cluster
 }
 
+type Limits struct {
```

Comment:
These limits are unrelated to LND and thus this struct should not be in the LND package.